frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemRow/dashboardListItemRow.styles.ts

```diff
@@ -47,17 +45,6 @@ export const Container = styled.div`
 		z-index: -1;
 	}
 
-	:hover {
-		border-color: ${({ theme }) => theme.palette.primary.contrast};
```

Comment:
I don't think we want to completely remove the hover right? It's specially the shadows  and the scaling?

@george-hadfield  can you be a bit more specific on what you're after here? Considering the line item is clickable I think you still want some sort of indication on hover ?